L1Trigger/TrackerTFP/interface/State.h

```diff
@@ -0,0 +1,166 @@
+#ifndef L1Trigger_TrackerTFP_State_h
+#define L1Trigger_TrackerTFP_State_h
+
+#include "L1Trigger/TrackerDTC/interface/Setup.h"
+#include "L1Trigger/TrackerTFP/interface/DataFormats.h"
+
+#include <vector>
+#include <numeric>
+
+
+namespace trackerTFP {
+
+  // 
+  class State {
+  public:
+    //
+    State(State* state);
+    // proto state constructor
+    State(const DataFormats* dataFormats, TrackKFin* track, int trackId);
+    // combinatoric state constructor
+    State(State* state, StubKFin* stub);
+    // updated state constructor
+    State(State* state, const std::vector<double>& doubles);
+    ~State(){}
+
+    //
```

Comment:
Add comment "Determine quality of completed state"

Reply:
done